index.html

```diff
@@ -56,9 +55,293 @@ <h2 class="connect">LET'S CONNECT</h2>
+                                  </div>
+                        
+                                  <div class="skill">
+                                    <div class="event12">
+                                      <div class="event7">
+                                        <h3 class="languages">Languages</h3>
+                                        <button type="submit" class="dropbtn">
+                                          <img src="./images/Icon-Chevron-Down-Larger.png" alt="" />
+                                        </button>
+                                      </div>
+                                      <ul class="language-list">
+                                        <li class="event9">
+                                          <div class="lang">
+                                            <img src="./images/Ellipse-1.png" alt="" />
+                                            <p>JavaSript</p>
+                                          </div>
+                                        </li>
+                                        <li class="event10">
+                                          <div class="lang">
+                                            <img src="./images/Ellipse-2.png" alt="" />
+                                            <p>HTML</p>
+                                          </div>
+                                        </li>
+                                        <li class="event11">
+                                          <div class="lang">
+                                            <img src="./images/Ellipse-3.png" alt="" />
+                                            <p>CSS</p>
+                                          </div>
+                                        </li>
+                                      </ul>
+                                    </div>
+                                    <div class="event-skill">
+                                      <div class="event17">
+                                        <h3>Frameworks</h3>
+                                        <button type="submit" class="dropbtn">
+                                          <img src="./images/Icon-Chevron-Right-Larger.png" alt="" />
+                                        </button>
+                                      </div>
+                                      <img
+                                        src="./images/Separator-Bottom.png"
+                                        alt=""
+                                        class="separator"
+                                      />
+                                      <div class="event18">
+                                        <h3>Skills</h3>
+                                        <button type="submit" class="dropbtn">
+                                          <img src="./images/Icon-Chevron-Right-Larger.png" alt="" />
+                                        </button>
+                                      </div>
```

Comment:
- [ ] As demanded by the project requirements, kindly make the list of skills categories to an unordered list. Your skills section should look like this.
```html
<ul>
  <li>
    Languages
    <ul>
      <li>HTML</li>
      ...
  </li>
  <li>Skills</li>
  <li>Frameworks</li>
</ul>
```


- [ ] Kindly add the icons to the right of the categories and also the line under the Frameworks and skills categories.
### Your design
![4](https://user-images.githubusercontent.com/94551484/192069508-0bce0123-d977-4a08-9e70-0aff5c945f16.png)
### Figma template
![f4](https://user-images.githubusercontent.com/94551484/192069522-a6ffaece-9399-4d6b-afd4-16bdfb7cbe8e.png)